### ISSUE TITLE
Tree: accessibility issues

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents#2465]
**Describe the bug**
There are couple of a11y issues with the tree view component:
1. Tree item is not announced by the screen reader please refer [here](http://veui5infra.dhcp.wdf.sap.corp:8080/sapui5-sdk-dist/#/entity/sap.m.Tree/sample/sap.m.sample.Tree)
2. Selected state of the tree item is announced by screen reader. please refer [here](http://veui5infra.dhcp.wdf.sap.corp:8080/sapui5-sdk-dist/#/entity/sap.m.Tree/sample/sap.m.sample.TreeSelection).
3. When user navigates to through the Tree, "tree view" is announced 2 times by screen reader.
4. Tree arrow to expand and collapse should have a tool tip. please refer [here](http://veui5infra.dhcp.wdf.sap.corp:8080/sapui5-sdk-dist/#/entity/sap.m.Tree/sample/sap.m.sample.Tree).

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Tree/
2. Start JAWS and navigate through tree.

**Isolated example**
https://sap.github.io/ui5-webcomponents/playground/components/Tree/

**Expected behavior**
1. Tree item should be announced.
2. Selected state should be announced.
3. "Tree vew" should be announced only once.
4.  Toggle arrow should have a tooltip as [here](http://veui5infra.dhcp.wdf.sap.corp:8080/sapui5-sdk-dist/#/entity/sap.m.Tree/sample/sap.m.sample.Tree)

**Screenshots**

**Context**
- UI5 Web Components version: 0.23.6

**Organization:** *(if applicable)*
Success Factor

**Priority:** *(optional)* *(Low, Medium, High or Very High)*
High

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-p

Consider this ACC issue for the BusyIndicator and this [PR](https://github.com/SAP/ui5-webcomponents/pull/2478) that tries to address the issues.

It is up to you if you want to create your own PR/change from scratch or review the existing one #2478. You can find the issue description and steps to reproduce in the issue description above.

BR,
ilhan

**Comment by lateefsofi** [comment]
@ilhan007 this ticket is closed long back but the issues are still there. Are you tracking these issues anywhere else?

**Comment by ilhan007** [comment]
No, it is closed with this PR https://github.com/SAP/ui5-webcomponents/pull/2636

Which version of ui5 web components are you using, because the fix might not be part of 0.23.X. It is part of 0.29.1, 0.29.2 versions

**Comment by lateefsofi** [comment]
I was checking [documentation](https://sap.github.io/ui5-webcomponents/playground/components/Tree/) for the fix. I believe web-components documentation might be updated with the latest release.

**Comment by ilhan007** [comment]
You can see the master version of the documentation
https://sap.github.io/ui5-webcomponents/master/playground/components/Tree/
